api/src/main/java/io/cloudevents/lang/Nullable.java

```diff
@@ -0,0 +1,19 @@
+package io.cloudevents.lang;
```

Comment:
Should we include license header?

Reply:
Since most license headers are broken, let's do it in a followup with formatting